CHANGELOG.md

```diff
@@ -1,58 +1,63 @@
+- `pairlist!` macro for generating `Pairlist` objects, e.g. for use in function calls [[#202]](https://github.com/extendr/extendr/pull/202)
+- `use_try_from` option for the `extendr` macro, which allows the use of any type that implements `TryInto<Robj>`/`TryFrom<Robj>`, e.g. `#[extendr(use_try_from = true)]` [[#222]](https://github.com/extendr/extendr/pull/222)
+- Large Rust integer types (`u32`, `u64` and `i64`) can now be converted to R's `numeric` type, which can handle large integer values [[#242]](https://github.com/extendr/extendr/pull/242)
+- `call!` macro, which can be used to call an R function whose name is provided as a string [[#238]](https://github.com/extendr/extendr/pull/238)
+- Implemented `TryFrom<Robj>` for a large number of Rust types [[#249]](https://github.com/extendr/extendr/pull/249), [[#258]](https://github.com/extendr/extendr/pull/258)
+- Support for `ALTREP` [[#250]](https://github.com/extendr/extendr/pull/250)
+- An `S4` struct, which wraps an S4 class in R [[#268]](https://github.com/extendr/extendr/pull/268)
+- Implemented `TryFrom<&ArrayBase> for Robj`, allowing `extendr`-annotated functions to return Arrays from the `ndarray` crate and have them automatically converted to R arrays. Note, even if 1D arrays are returned they will not be returned as vectors. [[#275]](https://github.com/extendr/extendr/pull/275)
+- `Rint`, `Rdouble`, `Rbool` and `Rcplx`: `ALTREP` wrapper classes for R vectors, which can be used without copying [[#274]](https://github.com/extendr/extendr/pull/274), [[#284]](https://github.com/extendr/extendr/pull/284), [[#301]](https://github.com/extendr/extendr/pull/301), [[#338]](https://github.com/extendr/extendr/pull/338), [[#350]](https://github.com/extendr/extendr/pull/350)
+- `Integers`, `Doubles`, `Strings`, `Logicals` and `Complexes`: `ALTREP` wrappers for vectors that allows for much larger vectors than native R [[#274]](https://github.com/extendr/extendr/pull/274), [[#284]](https://github.com/extendr/extendr/pull/284), [[#301]](https://github.com/extendr/extendr/pull/301), [[#338]](https://github.com/extendr/extendr/pull/338), [[#350]](https://github.com/extendr/extendr/pull/350)
+- `ExternalPtr`, a wrapper class for creating R objects containing any Rust object. [[#260]](https://github.com/extendr/extendr/pull/260)
+- Support for R graphics and graphics devices. These are locked behind the `graphics` feature flag, which is disabled by default. [[#279]](https://github.com/extendr/extendr/pull/279), [[#360]](https://github.com/extendr/extendr/pull/360), [[#373]](https://github.com/extendr/extendr/pull/373), [[#379]](https://github.com/extendr/extendr/pull/379), [[#380]](https://github.com/extendr/extendr/pull/380), [[#389]](https://github.com/extendr/extendr/pull/389)
+- Implemented `Deref` for vector types (Rint/Rfloat/Rbool/Rstr/Robj) to appropriately typed Rust slices [[#327]](https://github.com/extendr/extendr/pull/327)
```

Comment:
It is slightly misleading. We have vector wrappers like `Integers`, and we have scalar types like `Rint`. An item of `Integers` has type of `Rint`. So if I read the PR description correctly, it is basically dereferencing `Integers` into` &[Rint]`, mutable in some cases.

Reply:
Okay I've now updated my description. Feel free to re-review it.